the-collab-lab/tcl-63-smart-shopping-list
# Faq focus

## Description

Refactored accordion code to make the component accessible.

## Related Issue

None

## Acceptance Criteria

- [x] Accordion elements are tab-able
- [x] Each accordion element and its status (collapsed/uncollapsed) is announced by screen reader as a button.
- [x] Only one answer is revealed at a time.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓   | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓   | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Testing Steps / QA Criteria

To pull our branch locally, run git pull origin faq-focus
Next, run git checkout faq-focus
Install dependencies npm install
Launch the app with npm start

Check for the following:
1. Each accordion element is tab-able
2. Each accordion element is announced by screen reader as a button.
3. The screen reader announces the question and the answer (if open)
4. Only one answer is revealed at a time.
5. A dark outline is visible when an element is in focus.
